src/org.freedesktop.fwupd.xml

```diff
@@ -371,6 +371,42 @@
       </arg>
     </method>
 
+    <!--***********************************************************-->
+    <method name='GetApprovedFirmware'>
+      <doc:doc>
+        <doc:description>
+          <doc:para>
+            Gets the list of approved firmware.
```

Comment:
Might be worth expanding what approved firmware means here (it's not obvious unless you've reviewed this PR and know where it's going)

Reply:
`Gets the list of approved firmware that can be applied to devices`? I'm rubbish at prose like this.

Reply:
I think comments you've made elsewhere probably help.  How about this:

```
Gets the list of approved firmware that can be applied to the devices.  In an enterprise this has been typically configured by a domain administrator.
```